frontend/src/store/store.js

```diff
@@ -0,0 +1,24 @@
+import Vuex from 'vuex';
+import Vue from 'vue';
+
+import state from './state.js'
+import mutations from './mutations.js'
+import actions from './actions.js'
+import getters from './getters.js'
+
+import product from  './modules/product'
+
+Vue.use(Vuex)
+
+const store = new Vuex.Store({
+    state,
```

Comment:
You'll need to remove the state, mutations, actions, and getters here since they're redundant. The only thing that should be included at this level is the modules